src/modules/project/sagas.ts

```diff
@@ -88,19 +88,32 @@ export function* projectSaga(builder: BuilderAPI) {
 
   function* handleCreateProjectFromTemplate(action: CreateProjectFromTemplateAction) {
     const { template } = action.payload
-    const { title, description, onSuccess } = action.meta
+    const { title, description, sdk, onSuccess } = action.meta
 
-    const scene: Scene = {
-      sdk6: {
-        id: uuidv4(),
-        entities: {},
-        components: {},
-        assets: {},
-        metrics: EMPTY_SCENE_METRICS,
-        limits: EMPTY_SCENE_METRICS,
-        ground: null
-      },
-      sdk7: null
+    let scene: Scene
+
+    if (sdk === SDKVersion.SDK7) {
+      scene = {
+        sdk6: null,
+        sdk7: {
+          id: uuidv4(),
+          composite: '',
```

Comment:
The composite should not be an empty string otherwise the inspector would not work properly.. you can see how to generate a composite [here](https://github.com/decentraland/builder/blob/master/src/modules/inspector/utils.ts#L20-L56), you can skip the for loop part but at minimum the composite should have a Scene component with the amount of parcels that the user selected in the "size" step, you can copy that logic from the line 46 in the file on the link above